main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+    for player, info in players.items():
+        race = info["race"]
+        guild = info["guild"]
+
+        new_race = Race.objects.get_or_create(
+            name=race["name"],
+            description=race["description"]
+        )[0]
+
+        if guild:
```

Comment:
Your code will be better structured if you set `guild` as `None` first and then use `if` statement once without an extra `else` instruction. It is a generally accepted approach not to use too much branching if possible. In some programming languages to many if's can even make your code slower, so try to use less if possible.
```
guild = None
if guild := player_data.get("guild"):
     ...